src/assets/i18n/fr_FR.json

```diff
@@ -650,6 +650,16 @@
   "checkout.credit_card.user.lastname.missing.error": "Veuillez entrer le nom de famille du titulaire de la carte.",
   "checkout.credit_card.user.name.missing.error": "Veuillez entrer le nom du titulaire de la carte.",
   "checkout.credit_card.validityTime.error.notFound": "La durée de validité maximale du CVC n’a pas été trouvée.",
+  "checkout.desired_delivery_date.apply.button.label": "Appliquer",
+  "checkout.desired_delivery_date.error.date": "La date n’est pas valide. Veuillez utiliser le format correct (jj-mm-aaaa).",
+  "checkout.desired_delivery_date.error.max_date": "La date de livraison souhaitée ne peut être choisie car elle est trop éloignée dans le temps.",
+  "checkout.desired_delivery_date.error.min_date": "La date de livraison souhaitée ne peut pas être choisie car elle se situe dans le passé ou est inférieure à la date de livraison la plus proche possible.",
+  "checkout.desired_delivery_date.error.no_saturday": "La date de livraison souhaitée ne peut être choisie car les livraisons ne sont pas effectuées le samedi.",
+  "checkout.desired_delivery_date.error.no_sunday": "Votre date de livraison souhaitée ne peut pas être choisie car les livraisons ne sont pas effectuées le dimanche.",
+  "checkout.desired_delivery_date.label": "Date de livraison souhaitée",
+  "checkout.desired_delivery_date.note": "JJ-MM-AAAA",
```

Comment:
I don't think date format placeholders should be part of the translation file, as they can be derived from the standard shortDate format, which can be retrieved from there.

Maybe (in this case) add 3 general translation keys for replacing the letters:
```
 "common.date_format.letters.y": "A",
 "common.date_format.letters.M": "M",
 "common.date_format.letters.d": "J", 
``` 
And then use the shortDate format to transform it. (See other comment)